genesis/engine/entities/rigid_entity/rigid_entity.py

```diff
@@ -2443,6 +2443,46 @@ def get_qpos(self, qs_idx_local=None, envs_idx=None, *, unsafe=False):
         qs_idx = self._get_idx(qs_idx_local, self.n_qs, self._q_start, unsafe=True)
         return self._solver.get_qpos(qs_idx, envs_idx, unsafe=unsafe)
 
+    @gs.assert_built
+    def get_joints_anchor_pos(self, joints_idx_local=None, envs_idx=None, *, unsafe=False):
+        """
+        Returns anchor position of the entity's joints. This is the position of the joint in the world frame.
```

Comment:
This means nothing: "This is the position of the joint in the world frame."

Reply:
Here is the actual definition: https://github.com/Genesis-Embodied-AI/Genesis/blob/32904b3c1bc3a2840dae268d70509501e677193a/genesis/engine/entities/rigid_entity/rigid_joint.py#L104